mysql-test/t/percona_show_temp_tables_stress-master.opt

```diff
@@ -0,0 +1 @@
+$TOKUDB_OPT $TOKUDB_LOAD_ADD --loose-tokudb-check-jemalloc=0 --loose-tokudb-cache-size=512M --loose-tokudb-block-size=1M
```

Comment:
Yes, according to the comments in include/plugin.defs:

> # If a listed plugin is not found, the corresponding variables will be
> # set to empty, they will not be unset.

So if set to empty, they don't exist as options here and the positive results can be seen in the jenkins run. I actually tested the test locally to ensure that the TokuDB portion of the test is executed and/or not executed correctly if the plugin is not present by adding some extra 'echo's and changing the expected values of the table_counts to force the test to fail for both the TokuDB and non TokuDB cases.